modules/apps/remote-app/remote-app-service/src/main/resources/META-INF/portlet-model-hints.xml

```diff
@@ -10,9 +10,19 @@
 		<field name="userName" type="String" />
 		<field name="createDate" type="Date" />
 		<field name="modifiedDate" type="Date" />
-		<field localized="true" name="name" type="String" />
-		<field name="url" type="String">
+		<field name="customElementCSSURLs" type="String">
+			<hint-collection name="CLOB" />
+		</field>
+		<field name="customElementHTMLElementName" type="String">
+			<hint name="max-length">256</hint>
+		</field>
+		<field name="customElementURLs" type="String">
+			<hint-collection name="CLOB" />
+		</field>
+		<field name="iframeURL" type="String">
 			<hint name="max-length">1024</hint>
```

Comment:
Why 1024? I think IE support 2k

Reply:
We don't even support IE

![just-sayin](https://user-images.githubusercontent.com/905006/133286450-c61add46-408c-4b93-a61a-7ff813b49c11.gif)

Reply:
Because it was like that. We can change it (we usually use 2K as limit, for instance in the AMD loader and combo servlet). However, I prefer to do it in a different PR, so that we don't add more unrelated changes.

Reply:
This column (`iFrameURL`) has finally been changed to type `STRING` by providing `<hint-collection name="URL" />` in `portlet-model-hints.xml` so we don't need to do anything else :tada: .